api/peripheral_api/routes/LedSign.js

```diff
@@ -47,12 +38,10 @@ router.post('/updateSignText', async (req, res) => {
     logger.warn('/updateSignText was requested with an invalid token');
     return res.sendStatus(UNAUTHORIZED);
   }
-  const result = await SqsHandler.pushMessageToQueue(req.body);
-  if (result) {
-    res.sendStatus(OK);
-  } else {
-    res.sendStatus(BAD_REQUEST);
+  if (process.env.NODE_ENV !== 'production') {
+    return res.sendStatus(OK);
   }
+  await healthCheck();
```

Comment:
you also need to check the return value of these, if we are up return 200 else 500